blockchain/ethereum/contracts/erc721/presets/erc721presetminterpauserautoid deployer/contract preset/a146eef9-b442-42f6-b31c-c19bfc1ee593.json

```diff
@@ -0,0 +1,73 @@
+        "name": "data",
+        "description": "Receives contents of the store. San be set as parameter, or received as signal."
+      },
+      "0b28df10-b056-4666-ae14-2357d747f7b9": {
+        "type": "any",
+        "name": "read",
+        "description": "Receives a signal that triggers the contents being sent via `data` (output), or `not found` when has no content."
+      },
+      "5c2df9e2-9902-488b-9381-5474fb89b84b": {
+        "type": "{\n  \"name\": string,\n  \"symbol\": string,\n  \"baseTokenURI\": string\n}",
+        "name": "init parameters",
+        "description": "The arguments to be passed to the contract's constructor when deployed.\n"
+      }
+    },
+    "outputs": {
+      "e7a0df14-dc8c-41bd-8aba-367fdb468ce3": {
+        "type": "typeof `data`",
+        "name": "data",
+        "description": "Sends store contents."
+      },
+      "92b91df5-41d7-4e23-aeec-3f459099e659": {
+        "type": "null",
+        "name": "written",
+        "description": "Sends signal when contents were set in flow."
+      },
+      "e3876be5-e8c8-4983-b770-81fc8cdf5ff5": {
+        "type": "null",
+        "name": "not found",
+        "description": "Sends signal on read attempt while store has no content."
+      },
+      "db25229f-2c05-4616-b3ae-7799d5bed32d": {
+        "type": "string[]",
+        "name": "init parameters array"
+      }
+    }
+  },
+  "implementations": {
+    "es6": {
+      "cleanupFunction": "",
+      "dependencies": {},
+      "initFunction": "",
+      "inputFunctions": {
+        "543f8ad2-7755-44f6-bc9b-e6208c87b15a": "state.data = data;\noutputs.written(null, tag);",
+        "0b28df10-b056-4666-ae14-2357d747f7b9": "const value = state.data || params.data;\nif (value !== undefined)\n  outputs.data(value, tag);\nelse\n  outputs[\"not found\"](null, tag);"
+      }
+    },
+    "es6-node": {
+      "cleanupFunction": "",
+      "dependencies": {},
+      "initFunction": "",
+      "inputFunctions": {
+        "543f8ad2-7755-44f6-bc9b-e6208c87b15a": "state.data = data;\noutputs.written(null, tag);",
+        "0b28df10-b056-4666-ae14-2357d747f7b9": "const value = state.data !== undefined \n  ? state.data\n  : params.data;\nif (value !== undefined)\n  outputs.data(value, tag);\nelse\n  outputs[\"not found\"](null, tag);",
+        "5c2df9e2-9902-488b-9381-5474fb89b84b": "//  number-like keys are not allowed since they\n/// do not preserve the strict key order\nif (Object.keys(data).some((x)=>!isNaN(x))){\n  throw new Error(\"number-like key names are not allowed\");\n}\n\nconst initArray = Object.values(data).map((value)=>`${value}`);\noutputs[\"init parameters array\"](initArray, tag);\n"
```

Comment:
Converting a dictionary to an array using Object.values does not guarantee the order of the elements in the output array. Values will get into the array by order of creation that is up to the user thus non deterministic.

```
["name", "symbol", "baseTokenURI"].map((key) => data[key].toString())
```